common/changes/@uifabric/variants/variantThemeFunctionFix5.0_2018-08-27-16-17.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@uifabric/variants",
+      "comment": "cherry picked for 5.0",
```

Comment:
Nit: This cherry pick change file should probably be merged with the original one, i.e. the comment in the other change file should mention `Cherry pick of fixes to....`

Reply:
Ah, I was advised by someone else to make separate changefiles, which was what I was attempting to do here. I'll merge them in the future. Thanks!

Reply:
The main implication of this is that with 2 changefiles, when this 5.x change is published, you'll see 2 lines in the release notes and it wouldn't really read well/make sense :) Therefore, 1 combined change file is the recommended way so our release notes look clean and makes sense for downstream consumers.